lib/index.js

```diff
@@ -55,18 +58,18 @@ util.inherits(DataStore, EventEmitter)
  * @param {ConnectionOptions} options
  */
 DataStore.prototype.connect = function (options) {
-  this.connectionOptions = this.getConnectionOptions(options)
-  this.connectionString = this.constructConnectionString(this.connectionOptions)
+  const {name, options: databaseBlock} = this.getDatabaseOptions(options)
+  const connectionString = this.getConnectionString(name, databaseBlock)
 
-  debug('connect %s %o', this.connectionString, options)
+  debug('Connect %s %o', connectionString, options)
```

Comment:
Only if you're sure... the convention is lowercase, as the debug line starts with the whatever is specified when `debug` is required at the top of the file, e.g. `require('debug')('api-mongodb')`

Reply:
`api-mongodb: connect etc`